### PR TITLE
T6674: workflow: Add trigger to rebuild repo package

### PR DESCRIPTION
**Pull request by sever-sever**
## Change Summary

Add trigger to rebuild repo package
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [x] Other (please describe):

## Related Task(s)

 * https://vyos.dev/T6674

## Component(s) name

git-action
## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [ ] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: .github/workflows/trigger-rebuild-repo-package.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      REF: main  # Used for curl to trigger build package
+
+    steps:
+      - name: Set variables
+        run: |
+          echo "PACKAGE_NAME=$(basename ${{ github.repository }})" >> $GITHUB_ENV
+
+      - name: Trigger rebuild for ${{ env.PACKAGE_NAME }}
+        run: |
+          set -eux
```
Comment (c-po) [inline .github/workflows/trigger-rebuild-repo-package.yml]: Do we really want to expose all debug output?

**Review by c-po — CHANGES_REQUESTED** [review]

File: .github/workflows/trigger-rebuild-repo-package.yml (hunk 1 above)
Comment (sever-sever) [reply .github/workflows/trigger-rebuild-repo-package.yml]: Fixed!

**Review by dmbaturin — APPROVED** [review]

**Review by c-po — APPROVED** [review]

**Comment by c-po** [comment]
@Mergify backport circinus

**Comment by sever-sever** [comment]
@mergify backport circinus

**Pull request merged**